tests/unittests/OperatorTest.cpp

```diff
@@ -928,21 +928,35 @@ TEST_P(InterpAndCPU, ScatterAssignQuantized) {
 }
 
 TEST_P(Operator, QuantizeAndDequantize) {
-  Tensor inputs(ElemKind::FloatTy, {1, 4});
-  inputs.getHandle() = {1, 1.2f, 0.5f, 1.3f};
+  Tensor input1(ElemKind::FloatTy, {1, 4});
+  Tensor input2(ElemKind::FloatTy, {1, 4});
+  input1.getHandle() = {1, 1.2f, 0.5f, 1.3f};
+  input2.getHandle() = {1.9f, 5.2f, 3.5f, 6.3f};
 
   auto *A = mod_.createVariable(ElemKind::FloatTy, {1, 4}, "A",
                                 VisibilityKind::Public);
+  auto *B = mod_.createVariable(ElemKind::FloatTy, {1, 4}, "B",
+                                VisibilityKind::Public);
 
   auto qType = mod_.uniqueType(ElemKind::Int8QTy, {1, 4}, 0.05, -138);
-  auto *quantize = F_->createQuantize("quantize", A, qType);
-  auto *dequantize = F_->createDequantize("dequantize", quantize);
-  auto *result = F_->createSave("save", dequantize);
+  auto *quantizeA = F_->createQuantize("quantize", A, qType);
+  auto *quantizeB = F_->createQuantize("quantize", B, qType);
+  auto *int8Add = F_->createAdd("int8Add", quantizeA, quantizeB);
+  auto *dequantize = F_->createDequantize("dequantize", int8Add);
+  auto *int8Result = F_->createSave("int8Save", dequantize);
```

Comment:
a bit misleading name as the result in fact fp32 here.

Reply:
removed the prefix